ui-library-filesharing-chat-composite/api/UploadFileToAzureBlobStore/index.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { AzureFunction, Context, HttpRequest } from '@azure/functions';
+import * as multipart from 'parse-multipart';
+import HTTP_CODES from 'http-status-enum';
+
+const azureStorageConnectionString = process.env['azureStorageConnectionString'];
+let storageAccountName;
+try {
+  storageAccountName = azureStorageConnectionString.match(/AccountName=([^;]+)/)[1];
+} catch {
+  console.error('Failed to parse Azure storage name from connection string');
```

Comment:
context.error

Reply:
This is correct. Console errors will be shown to the user during local development. Context errors won't.